src/components/common/BookCard/BookCard.js

```diff
@@ -14,29 +14,29 @@ const BookCard = ({ children, book }) => {
           {children || (
             <>
               <div className="text-black">
-                <Typography variant="h6" className="BookCard__title">
+                <Typography variant="h6" className="BookCard__title text-10">
                   {book.title}
                 </Typography>
                 <div className="d-flex">
                   {book.categories.map((item, idx) => (
```

Comment:
for every object have  ?  before using their properties I dont remember the JS term but like this
book?.categories?.map
so even if book is undefined if api fails the app wont crash

Reply:
Yes, it's called optional chaining. But I don't understand why you would require it here.